ReleaseProcessAutomation.Tests/IntegrationTests/ReleaseFromHotfixTests.cs

```diff
@@ -106,9 +106,45 @@ public void ReleaseNewPatch_FromHotfix_ToSupport ()
     TestConsole.Input.PushTextWithEnter("1.1.1");
     //Get next release version from user for jira
     TestConsole.Input.PushTextWithEnter("1.1.2");
+    //Confirms to create new support branch
+    TestConsole.Input.PushTextWithEnter("n");
 
     var act = Program.Main(new[] { "Release-Version" });
+    
+    AssertValidLogs(correctLogs);
+    Assert.That(act, Is.EqualTo(0));
+  }
 
+  [Test]
+  public void ReleaseNewPatch_FromHotfixWithNewSupportBranch_ToSupportCreatesNewSupportBranch ()
+  {
+    
+    var correctLogs =
+        @"*  (hotfix/v1.1.2)Update metadata to version '1.1.2'.
+          | *  (hotfix/v1.2.0)Update metadata to version '1.2.0'.
+          |/  
+          *    (HEAD -> support/v1.1, tag: v1.1.1, origin/support/v1.1, support/v1.2) Merge branch 'release/v1.1.1' into support/v1.1
+          |\  
+          | *  (origin/release/v1.1.1, release/v1.1.1)Update metadata to version '1.1.1'.
+          |/  
+          *  (master, hotfix/v1.1.1)ConfigAndBuildProject
+          *  (origin/master)Initial CommitAll
+          ";
+
+    correctLogs = correctLogs.Replace(" ", "").Replace("\r", "");
+    ExecuteGitCommand("checkout -b support/v1.1");
+    ExecuteGitCommand("checkout -b hotfix/v1.1.1");
+    ExecuteGitCommand("commit -m Commit on hotfix --allow-empty");
+
+    //Get release version from user
+    TestConsole.Input.PushTextWithEnter("1.1.1");
+    //Get next release version from user for jira
+    TestConsole.Input.PushTextWithEnter("1.1.2");
+    //Confirms to create new support branch
+    TestConsole.Input.PushTextWithEnter("y");
+
+    var act = Program.Main(new[] { "Release-Version" });
+    
     AssertValidLogs(correctLogs);
     Assert.That(act, Is.EqualTo(0));
   }
```

Comment:
This scenario seems weird:
why is there both a support v1.2 and a support v1.1 (including hotfixes) in the expectation?

Reply:
As far as I understood the docs, when creating a new support branch at the release of a hotfix, we want the old branch to continue running with a new hotfix version. 

Let's assume we are on hotfix/v1.1.1 and we release this to support/v1.1, we also want to create a new support branch from that release. 
That means we have a new support/v1.2 with a new hotfix/v1.2.0 aswell as the old support/v1.1 with a new hotfix/v1.1.2

I'm not sure this is the behaviour that we want, but I think it's the behaviour outlined in the pptx.

Reply:
> That means we have a new support/v1.2 with a new hotfix/v1.2.0 aswell as the old support/v1.1 with a new hotfix/v1.1.2
This interpretation is incorrect. The support/v1.2 can be created as a fork from support v1.1, but only on demand, not automatically.

Reply:
That is the functionality in the test though. 
The script asks the user if he wants to create a new support branch, which is confirmed in this test on line 144. 

The test above this one oultines what happens when we do not want to create a new support branch. It merely creates the next hotfix version.